Pikipick/Pikipick/LocalNetworkPrivacy.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  LocalNetworkPrivacy.swift
+//  Pikipick
+//
+//  Created by woo0 on 2022/07/20.
+//
+
+import Foundation
+import Network
+import UIKit
+
+class LocalNetworkPrivacy : NSObject {
```

Comment:
```suggestion
final class LocalNetworkPrivacy : NSObject {
```

더이상 상속되지 않는 class인 것을 알기 쉽도록 final을 붙이는것을 추천드려욥

Reply:
넵! 감사합니다!